### ISSUE TITLE
Several issues with rooms

### ISSUE THREAD

**Issue by joto** [mstock/RoomAssets-App-Sync#1]
I have some issues with room names having non-ascii characters, in this case German umlauts. Testing with event 'fossgis2024' 
the following rooms should be available but the first 4 do not show up:

(There is also the additional issue that the "language" for the room name is not 'en' as it is hardcoded in the script, but 'de- formal'. Maybe it would be better to just autodetect what rooms are there and download all, or at least have that as an option? Or maybe have a `--detect-rooms` option or so?) 

```
    --room 'Hörsaal 1 (Audimax 1)' \
    --room 'Hörsaal 2 (Ditze H016)' \
    --room 'Hörsaal 3 (K0506/ Audimax 2)' \
    --room 'Hörsaal 4 (A0.13)' \
    --room 'Expert:innen (H.04)' \
    --room 'Anwendertreffen/BoF1 (H.09)' \
    --room 'Anwendertreffen/BoF2 (H.08)' \
    --room 'Workshop 1 (D.013)' \
    --room 'Workshop 2 (D.011)' \
    --room 'Workshop 3 (D.010)'
```

**Comment by mstock** [comment]
Thanks for your feedback! I just pushed two commits which should resolve most if not all the issues you found. The `--room` parameter is now optional, but German umlauts should actually work, too. With these changes, assets for all rooms should be downloaded by default, so `--room` is just an optional filter now, which simplifies the command. I've also removed the hard-coded `en` and introduced a `--language` parameter, which is even optional if there is only one conference language configured in Pretalx.
